### ISSUE TITLE
NodeWrapper events (onMouseEnter, onMouseMove, onMouseLeave) not being emitted

### ISSUE THREAD

**Issue by jayrchamp** [bcakmakoglu/vue-flow#253]
# NodeWrapper events (onMouseEnter, onMouseMove, onMouseLeave) not being emitted

In the file NodeWrapper.vue, it seems like the following methods: onMouseEnter, onMouseMove and onMouseLeave will never be able to trigger the event since "dragging" is a ref object and will always result to be true. 

It should be changed to "dragging.value" so that it is correctly evaluated.

Example:
```
const onMouseEnter = (event: MouseEvent) => {
  if (!dragging.value) {
    emit.mouseEnter({ event, node, connectedEdges: getConnectedEdges([node], edges) })
  }
}
```

Thank you

**Comment by bcakmakoglu** [comment]
Nice catch, thanks. 
Ill merge the PR and release the fix during the day 👍

**Comment by bcakmakoglu** [comment]
@jayrchamp released with `0.4.33`
